sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -17,7 +17,9 @@
     List,
     TYPE_CHECKING,
     cast,
+    overload
 )
+import warnings
```

Comment:
I don't recall that we use `warnings` in our SDKs -- we usually use logger.warning if I remember correctly.

Reply:
we use warnings.warn in a few places, for ex: https://github.com/Azure/azure-sdk-for-python/blob/f90387f693220527a9f8eb2408749bac94bfc458/sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py#L307

I can change it, but I think this works too.

Reply:
aligning with rest of EH by using _LOGGER.warning